src/commands/give-up/index.ts

```diff
@@ -18,29 +19,24 @@ const giveUp = async (
   context: Context,
   challengeIssueService: IChallengeIssueService
 ) => {
-  const issue = context.issue();
-  const issueResponse = await context.github.issues.get({
-    owner: issue.owner,
-    repo: issue.repo,
-    issue_number: issue.number,
-  });
-  const { data } = issueResponse;
+  const issueKey = context.issue();
```

Comment:
Maybe we can use `issuekey` to distinguish the specific issue data and the identifier of the issue. I find a similar explanation.

https://support.atlassian.com/jira-software-cloud/docs/what-is-an-issue/#Workingwithissues-Issuekeys